api/v1alpha1/packagebundle_types.go

```diff
@@ -109,14 +110,15 @@ type PackageBundleStatus struct {
 	State PackageBundleStateEnum `json:"state"`
 }
 
-// +kubebuilder:validation:Enum={"available","ignored","invalid"}
+// +kubebuilder:validation:Enum={"available","ignored","invalid","upgrade required"}
 // PackageBundleStateEnum defines the observed state of PackageBundle.
 type PackageBundleStateEnum string
 
 const (
-	PackageBundleStateAvailable PackageBundleStateEnum = "available"
-	PackageBundleStateIgnored   PackageBundleStateEnum = "ignored"
-	PackageBundleStateInvalid   PackageBundleStateEnum = "invalid"
+	PackageBundleStateAvailable       PackageBundleStateEnum = "available"
+	PackageBundleStateIgnored         PackageBundleStateEnum = "ignored"
+	PackageBundleStateInvalid         PackageBundleStateEnum = "invalid"
+	PackageBundleStateUpgradeRequired PackageBundleStateEnum = "upgrade required"
```

Comment:
This might be confusing because upgrades are tracked on the PBC.

Reply:
"controller upgrade required" maybe?